### ISSUE TITLE
Keyboard navigation on mobile devices

### ISSUE THREAD

**Issue by bglover** [FrDH/mmenu-js#1259]
Hello,

We recently had a firm perform an accessibility review on our website. One of the things they called out was the off-canvas menu because it "is not accessible for keyboard or screen reader users.". In the documentation for the [keyboard navigation](https://mmenujs.com/docs/addons/keyboard-navigation.html) addon it states

> Note that the "keyboardNavigation" add-on is disabled for touchscreens to prevent an virtual keyboard to appear when opening the menu.

I can see where this is disabled in [src/addons/keyboardnavigation/mmenu.keyboardnavigation.ts](https://github.com/FrDH/mmenu-js/blob/master/src/addons/keyboardnavigation/mmenu.keyboardnavigation.ts#L13-L17). If I comment that out and build mmenu myself, keyboard navigation on mobile devices works like a charm.

We build mmenu ourselves so I'd like to just comment out that condition, but before I go this route I want to find out more about the code comment
```javascript
// Keyboard navigation on touchscreens opens the virtual keyboard :/
// Lets prevent that.
```

With the above condition commented out and with keyboard navigation working on mobile devices, I haven't experienced that issue on my own mobile device (Android 11, Chrome), and I haven't been able to replicate the issue using an iPhone 
 (using Safari) either. Is this something that can occur only on certain devices, or is that to catch a certain edge case? How can I go about replicating that behavior (where the keyboard navigation addon opens the virtual keyboard)?

Thanks!

**Comment by FrDH** [comment]
To be honest I'm not sure what the exact issue was with virtual keyboards. If disabeling that line works for you, please keep doing it.
In the next major release keyboard navigation will no longer be an add-on, it will be implemented where it is needed.
Also, there is no longer a touch-device check :)

**Comment by bglover** [comment]
Sounds good. Thanks for getting back to me so quickly! I'll keep my eye out for the next version.
